### ISSUE TITLE
ML/AI: Custom Dataloader using pytorch 

### ISSUE THREAD

**Issue by sakethbachu** [IEEESFIT1/OpenOctober#121]
Custom Data loaders are necessary for a deep learning task, this code will easily load data with the labels and also will do the necessary pre-processing.

**Comment by sakethbachu** [comment]
@orvil1026
